actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -20,12 +20,15 @@ def process_action(event)
       info do
         payload   = event.payload
         additions = ActionController::Base.log_process_action(payload)
-
         status = payload[:status]
```

Comment:
Nitpick: After removing the blank line I would format `status` to the same level as`additions` as we seem to follow that style across this file.
```ruby
payload   = event.payload
additions = ActionController::Base.log_process_action(payload)
status    = payload[:status]
```

Reply:
I went the other way. Evening the space makes searching harder so I'm not a huge fan.

Reply:
Great! Thanks for caring about this and for the PR ❤️